### DISCUSSION TITLE
n8n.io

### DISCUSSION THREAD

**Discussion by cryptosmithio** [TriliumNext/Trilium#2139] [Ideas]
I am thinking of integrating with n8n if I can get the restful api worked out. Once working with n8n all other integrations should be pretty easy. 

Anyone else working with the rest api?

**Comment by zadam** [comment]
Hi, Trilium itself doesn't have any suitable REST APIs for this purpose (yet). But [custom request handlers](https://github.com/zadam/trilium/wiki/Custom-request-handler) allow you to build the API you need ...
